code/game/turfs/simulated/floor.dm

```diff
@@ -95,7 +100,13 @@
 	return
 
 /turf/open/floor/proc/update_icon()
+	..()
```

Comment:
Why are we now running the super proc ..()

then running it a second time on 105, and catching the return?

DM me if stuck

Reply:
Doesn't look like it needs return 0 for any reason, so this should be fine.